docker/zap-api-scan.py

```diff
@@ -417,6 +421,7 @@ def main(argv):
                     # Dont bother checking the result - this will fail for pscan rules
                     zap.ascan.set_scanner_alert_threshold(id=scanner, alertthreshold='OFF', scanpolicyname=scan_policy)
 
+        if not baseline:
         zap_active_scan(zap, target, scan_policy)
```

Comment:
zap_active_scan will need to be indented - python will complain otherwise wont it?